### ISSUE TITLE
How to draw a path using InkCanvas in Touch?

### ISSUE THREAD

**Issue by SanthiyaArulsamy** [microsoft/Win2D#737]
Hi,

I am having an InkCanvas inside the Grid control and I am trying to set the InkPresenter’s InputDeviceTypes property value as Touch or Mouse in the Grid PointerEntered event and I changed the InkPresenter’s InputDeviceTypes to None in the InkPresenter StrokeCollected event.

So, I can enable the drawing on a pointer to enter and disable the drawing after collected the strokes. This is my requirement.

It is working properly on using the mouse, but it is not working on Touch. I can draw the path in touch in the first time only. The second time it is not drawing.

Please help me how to resolve this issue.

Code snippet:

Xaml:

<Grid x:Name="grid" PointerEntered="Grid_PointerEntered" PointerMoved="Grid_PointerMoved" 
          PointerPressed="Grid_PointerPressed">
        <Image Source="Buldingimage1.jpeg"/>
        <InkCanvas x:Name="canvas"/>
    </Grid>
Events

   private void InkPresenter_StrokesCollected(InkPresenter sender, InkStrokesCollectedEventArgs args)
    {
        canvas.InkPresenter.InputDeviceTypes = Windows.UI.Core.CoreInputDeviceTypes.None;
    }

    private void Grid_PointerEntered(object sender, PointerRoutedEventArgs e)
    {
        canvas.InkPresenter.InputDeviceTypes =
       Windows.UI.Core.CoreInputDeviceTypes.Mouse |
       Windows.UI.Core.CoreInputDeviceTypes.Pen | Windows.UI.Core.CoreInputDeviceTypes.Touch;
    }
Sample: https://drive.google.com/open?id=1wT9MvnJyXWR91ifUV3tJZJIaNluhfyK_

Thanks, Santhiya A

**Comment by shawnhar** [comment]
This seems like more of a XAML input handling question rather than a Win2D topic.  It's outside my area of expertise, but I wouldn't generally expect a need to be changing input device types on the fly like this.  Why are you setting that to None?

**Comment by SanthiyaArulsamy** [comment]
I want to get the finger count when drawing. I have tried the ActiveCustomDrying method and set SetPredefinedConfiguration as Multiple pointers to get the finger count. I can get the finger count but I can't use the StrokeContainer because it's valued as null when using ActiveCustomDrying. I need StrokeContainer. So I tried to like this. My requirement is to draw the path in a single finger and zoom the image in double finger.
